R/uvhydrograph-render.R

```diff
@@ -102,9 +102,10 @@ createSecondaryPlot <- function(data, month){
   table <- NULL
   status_msg <- NULL
   
-  isSecondarySeries <- any(grepl("downsampledSecondarySeries", names(data)))
+  isReferenceSeries <- any(grepl("downsampledReferenceSeries", names(data)))
+  isUpchainSeries <- any(grepl("downsampledUpchainSeries", names(data)))
```

Comment:
@thongsav-usgs if there is no reference timeseries, but there is an upchain timeseries do we plot the data? That might be a question for Laura.